resilience4j-spring/src/main/java/io/github/resilience4j/bulkhead/configure/BulkheadAspect.java

```diff
@@ -99,6 +116,27 @@ private Object handleJoinPoint(ProceedingJoinPoint proceedingJoinPoint, io.githu
 		}
 	}
 
+	/**
+	 * handle the asynchronous completable future flow
+	 *
+	 * @param proceedingJoinPoint AOPJoinPoint
+	 * @param bulkhead            configured bulkhead
+	 * @param methodName          bulkhead method name
+	 * @return CompletionStage
+	 * @throws Throwable
+	 */
+	private Object handleJoinPointCompletableFuture(ProceedingJoinPoint proceedingJoinPoint, io.github.resilience4j.bulkhead.Bulkhead bulkhead, String methodName) throws Throwable {
```

Comment:
i guess handleAsync would give a feeling it can handle other types of Async like Futures..ect, the method name here is more specific to what it support which CompletableFuture only .

Reply:
ok, can use the same method name in CircuitBreakerAspect as well?

Reply:
On it now